ui-community/src/components/layouts/admin/components/tickets-list.tsx

```diff
@@ -66,12 +80,31 @@ export const ServiceTicketsList: React.FC<any> = (props) => {
   ];
   console.table(props.data?.serviceTicketsResults);
   return (
-    <div>
-      <Table
-        columns={columns}
-        dataSource={props.data?.serviceTicketsResults as any}
-        rowKey={(record: any) => record.id}
-      />
-    </div>
+    <Content>
```

Comment:
Since we are using Cognitive Search, we can not use the default table paginations. Instead, we need to use two separate instances of fully controlled Pagination components.